introduction.md

```diff
@@ -1,2 +1,6 @@
 Hi This is Vishal Gaikwad.
 My favorite Code School path - JavaScript
+* Javascript
+* Git
+* Html
+* Css
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?